src/components/pick-list/shared-list-render.tsx

```diff
@@ -24,14 +25,16 @@ export const List: FunctionalComponent<{ props: ListProps } & DOMAttributes<any>
     dataForFilter,
     handleFilter,
     filterPlaceholder,
-    setFilterEl
+    setFilterEl,
+    dragEnabled
   },
   ...rest
 }): VNode => {
   const defaultSlot = <slot />;
   return (
     <Host aria-busy={toAriaBoolean(loading)} role="menu" {...rest}>
       <section>
+        {dragEnabled ? <span aria-live="assertive" class="assistive-text" tabindex="-1" /> : null}
```

Comment:
Was there discussion on the usage of `assertive` for `aria-live`? Usually its restricted to time sensitive or critical notifications [[Resource](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)]. 

If you agree we don't need to dismiss previous verbiage to screen readers, can we change to `aria-live="polite"`?

Reply:
`polite=assertive` is required for the assistive text to be able to announce the instructions as it is not the focused element when the dragging /sorting is in happening. if we use `aria-live = polite` then the instructions will be placed in queue and might not be announced as expected.

Reply:
Ah, that's right, I think we discussed this earlier as well. Sounds like a great use case for `aria-live="assertive"`. Thanks @anveshmekala!